src/components/CardContainer/CardContainer.tsx

```diff
@@ -1,27 +1,26 @@
 import React, { useEffect } from "react";
 import Card from "../Card/Card";
-import data from "../../mockData/mockData.json";
 import { CardContainerStyle } from "./CardContainer-style";
 import { ICard } from "../../Utils/Types";
-import { APIkey1 } from "../../Utils/Constances";
-import { fetchFromApi } from "../../store/reducers/articlesReducer";
+import { API_KEY } from "../../store/Utils/storeConstances"
+import { fetchFromApi } from "../../store/Utils/index";
 import { useAppDispatch, useAppSelector } from "../../store/store";
-
+import { EVERYTHING_STRING, TOP_HEADLINES_STRING } from "../../store/Utils/storeConstances"
 
 const CardContainer = () => {
 
   const dispatch = useAppDispatch();
   
   useEffect(()=>{
-    dispatch(fetchFromApi(`https://newsapi.org/v2/everything?q=bitcoin&apiKey=${APIkey1}`));
+    dispatch(fetchFromApi(`${EVERYTHING_STRING}` + `${API_KEY}`));
```

Comment:
we use `` to combine between text (string) and evaluation of functions or variables. Here  you use only variables so you can just write EVERYTHING_STRING + API_KEY.